data/p5js/mountain-view/index.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: p5js
+authors: [Edmundo Mejia Galindo, Julia Hilt]
```

Comment:
authors and authors-url does not work at the moment with normal sketches. I'll need to rework that to be like in the projects. Problem is all the other sketches don't have multiple authors.  
can you:  

1. add a `author` and a `author-url` field.  
2. change the `authors: [` to a list like defined [here](https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#front-matter)   
3. The guide on how to contribute sketches as to many [VALUE] things. It creates misunderstandings. I'll change that ASAP.